istioctl/pkg/validate/validate.go

```diff
@@ -171,16 +280,24 @@ func NewValidateCommand() *cobra.Command {
 	var referential bool
 
 	c := &cobra.Command{
-		Use:     "validate -f FILENAME [options]",
-		Short:   "Validate Istio policy and rules",
-		Example: `istioctl validate -f bookinfo-gateway.yaml`,
-		Args:    cobra.NoArgs,
+		Use:   "validate -f FILENAME [options]",
+		Short: "Validate Istio policy and rules",
+		Example: `
+		# Validate bookinfo-gateway.yaml
+		istioctl validate -f bookinfo-gateway.yaml
+		
+		# Validate current deployments within the cluster
```

Comment:
`Validate current deployments in the default namespace` - the command is not cluster wide, and cluster-wide commands were deprecated in K8s 1.11 and removed from K8s in 1.14 iirc.

Reply:
changed to ```# Validate current deployments under 'default' namespace within the cluster
		kubectl get deployments -o yaml |istioctl validate -f -```